soroban-env-macros/src/synth_wasm_expr_type.rs

```diff
@@ -117,8 +117,24 @@ pub fn generate(file_lit: LitStr) -> Result<TokenStream, Error> {
     let mut all_subty_fns: BTreeSet<String> = BTreeSet::new();
     let mut fns_by_type = BTreeMap::<String, Vec<crate::Function>>::new();
 
+    let mut allowed_fns: BTreeSet<String> = BTreeSet::new();
+
     for m in root.modules.iter() {
+        if m.name == "t" {
+            continue;
+        }
         for f in m.functions.iter() {
+            if let Some(proto) = f.min_supported_protocol {
+                if proto > 20 {
```

Comment:
Why is these checks done against 20?

Reply:
Oh, it's actually _supposed_ to be skipping anything that's not supported on all protocols, but I think the second condition is wrong and anyways it can be simplified. I'll fix. It's only used by the expression-fuzzer anyway. Which I forgot to re-repair here (the repair is in the other PR, this got copied in by accident). I'll complete the repair here.